apps/client/src/api/createPayment.tsx

```diff
@@ -0,0 +1,7 @@
+// src/api/debts.ts
+import axios from "axios";
+
+export const postCreatePayment = async (data: { apartmentId: string, fromId: string, toId: string, amount: number }) => {
+    const response = await axios.post("/api/payment/create-payment", data);
```

Comment:
```suggestion
    const response = await axios.post<ResponseType>("/api/payment/create-payment", data);
```

And also maybe the body data, import from types as well , so the backend can use too